homeassistant/loader.py

```diff
@@ -1550,6 +1550,20 @@ def __init__(self, hass: HomeAssistant) -> None:
     def __getattr__(self, helper_name: str) -> ModuleWrapper:
         """Fetch a helper."""
         helper = importlib.import_module(f"homeassistant.helpers.{helper_name}")
+
+        # Local import to avoid circular dependencies
+        from .helpers.frame import report  # pylint: disable=import-outside-toplevel
+
+        report(
+            (
+                f"accesses hass.helpers.{helper_name}."
+                " This is deprecated and will stop working in Home Assistant 2024.10, it"
```

Comment:
Why do we need to add it to the April release? May would be ok too, right?

Reply:
Yea that is ok too. Changed the version